### ISSUE TITLE
Union Types don't generate mapper types inside ResolversTypes

### ISSUE THREAD

**Issue by SirensOfTitan** [dotansimha/graphql-code-generator#1758]
Repository with repro generation: https://github.com/SirensOfTitan/gql-codegen-union-mapper

Essentially if I have a union with a mapper type:
`union MovieLike = Movie | Book`

```yaml
mappers:
  Movie: ./types#MovieEntity
```

Actual Result Result: 
```typescript
export type ResolversTypes = {
  String: Scalars["String"];
  Boolean: Scalars["Boolean"];
  Movie: MovieEntity;
  ID: Scalars["ID"];
  Int: Scalars["Int"];
  Book: Book;
  MovieLike: MovieLike;
};

export type MovieLike = Movie | Book;

```

Expected result:
```typescript
export type ResolversTypes = {
  String: Scalars["String"];
  Boolean: Scalars["Boolean"];
  Movie: MovieEntity;
  ID: Scalars["ID"];
  Int: Scalars["Int"];
  Book: Book;
  // e.g. MovieLike = MovieEntity | Book
  MovieLike: ResolversTypes["Movie"] | ResolversTypes["Book"];
};
```

**Comment by dotansimha** [comment]
Thanks @SirensOfTitan . Fixed in https://github.com/dotansimha/graphql-code-generator/pull/1759

**Comment by dotansimha** [comment]
Fixed in 1.1.1 🎉
